css/module.css

```diff
@@ -0,0 +1,84 @@
+/*--------------App Module-------------------*/
+/*This following style rule is for the stylesheet */
+.component-section {
+  padding: 3em;
+  border: .1em solid rgba(0, 0, 0, 0.2);
+}
+
+/*--------------Header Module-------------------*/
+.app-header {
+  background-color: rgba(54, 48, 49, 1.0);
+  padding: 1em 0 0 1em;
+  border-bottom: .2em solid rgba(178, 80, 36, 1.0);
+}
+
+.app-header h1 {
+  display: inline-block;
+  vertical-align: top;
+  font-size: 3em;
+  font-weight: 800;
+}
+
+.page-header {
+  padding: 5em;
+  background-color: white;
+  border-bottom: .1em solid rgba(54, 48, 49, 0.2);
+}
+
+.page-header h2 {
```

Comment:
Following the rules of single responsibility, this would make the styles dependent on its parent. Instead of using the descendant selector, consider creating a class specifically for the h2. Something like:
```
.page-header_sub-heading {}
```